### ISSUE TITLE
Bazecore 1.4 rc 2 Superkeys Editor error

### ISSUE THREAD

**Issue by kyleGrealis** [Dygmalab/Bazecor#734]
**Describe the bug**
After connecting my boards, selecting Superkeys Editor displays this image:
![image](https://github.com/Dygmalab/Bazecor/assets/84798289/9a649fde-eb86-456f-995e-8f7b50fe3320)

**To Reproduce**
1. Go to 'Connect'
2. Click on 'Superkeys Editor (Beta)'

**Expected behavior**
Edit super keys.

**Screenshots**
If applicable, add screenshots to help explain your problem. If possible, please also include a log from the "developer console". You can access it by pressing `Control + Shift + I`. When it appears, right click inside it, and select "Save as", and attach the saved file to this report.

**Desktop (please complete the following information):**
 - OS: Windows
 - Bazecor Version: 1.4.0-rc.2

**Additional context**
N/A

**Comment by kyleGrealis** [comment]
This error is replicated on Ubuntu 22.04 with the same RC version.

**Comment by dlnilsson** [comment]
got the same issue on Windows with 1.4.0-rc.2 using firmware v1.2.6-beta

![image](https://github.com/Dygmalab/Bazecor/assets/1218385/162f785d-b1cd-4d45-995d-e49aa094e598)

**Comment by alexpargon** [comment]
Hey there @dlnilsson @kyleGrealis! can any of you share your layers? I tried to reproduce this issue by copying just the superkeys but it did not reproduce.

**Comment by zanza00** [comment]
I have the [same problem](https://discord.com/channels/895297925578637362/1215237487232815154), let's continue here

log after importing a backup
[Message (2).txt](https://github.com/Dygmalab/Bazecor/files/14538479/Message.2.txt)

backup of layers
[Defy.json](https://github.com/Dygmalab/Bazecor/files/14538487/Defy.json)

**Comment by dlnilsson** [comment]
> Hey there @dlnilsson @kyleGrealis! can any of you share your layers? I tried to reproduce this issue by copying just the superkeys but it did not reproduce.

- https://github.com/dlnilsson/defy-backups 
- https://github.com/dlnilsson/defy-backups/blob/1ed205dedeb89394b50a963c89a1eca1a3d90b45/Defy/cc450da6d4aaead/20240309113723-dln%20neuron.json

**Comment by alexpargon** [comment]
Thanks Guys! I found out the root cause, it's the electron store, there is a parameter that is not being settled and is breaking when you try to read it, it's the one that saves the selection of standard/single view.

This value is not properly set after a clean install and it's failing as I transformed the superkeys view to Typescript, so I've been migrating the whole electron-storage library to a typescript schema and creating types for definition and access to it.
